src/main/java/marquez/client/models/DatasetMeta.java

```diff
@@ -36,9 +37,15 @@
 public abstract class DatasetMeta {
```

Comment:
Let's handle the case where the _fields_ or _tags_ can be null by defining the constructor as discussed [here](https://github.com/MarquezProject/marquez-java/pull/58/files#r374504829) (I left out all other fields for readability):

```java
public DatasetMeta(
  @Nullable final List<Field> fields,
  @Nullable final List<String> tags) {
  this.fields = ImmutableList.copyOf(new ArrayList<>(fields)); 
  this.tags = ImmutableList.copyOf(new ArrayList<>(tags)); 
}
```

Reply:
Done